api/login_api.py

```diff
@@ -28,7 +28,9 @@ class LoginAPI(basehandlers.APIHandler):
   """Create a session using the credential generated by Google Sign-In."""
 
   def do_post(self):
-    token = self.get_param('credential')
+    # TODO(jrobbins): Remove id_token after next deployment.
+    token = (self.get_param('id_token', required=False) or
+             self.get_param('credential'))
     message = "Unable to Authenticate"
```

Comment:
Let make it more actionable. "Unable to Authenticate and please log in again"?

Reply:
Done.